pkgs/top-level/aliases.nix

```diff
@@ -738,4 +738,7 @@ mapAliases ({
   /* Cleanup before 21.03 */
   riot-desktop = throw "riot-desktop is now element-desktop!";
   riot-web = throw "riot-web is now element-web";
+
+  ant-dracula-theme = throw "ant-dracula-theme is now dracula-theme";
```

Comment:
Can this mention that the theme name has changed. For example I will have to change my config [here](https://github.com/alexarice/dotfiles/blob/f10a92b45730adf87b1f39e7c891ab25cfe15fc8/dotfiles/gtk.nix#L13)